[2024-10-05T15:00:12.601+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: pdf_extraction_with_dochorizon.extract_with_dochorizon manual__2024-10-05T15:00:07.704358+00:00 [queued]>
[2024-10-05T15:00:12.611+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: pdf_extraction_with_dochorizon.extract_with_dochorizon manual__2024-10-05T15:00:07.704358+00:00 [queued]>
[2024-10-05T15:00:12.611+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2024-10-05T15:00:12.612+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2024-10-05T15:00:12.612+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2024-10-05T15:00:12.627+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): extract_with_dochorizon> on 2024-10-05 15:00:07.704358+00:00
[2024-10-05T15:00:12.632+0000] {standard_task_runner.py:55} INFO - Started process 14798 to run task
[2024-10-05T15:00:12.634+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pdf_extraction_with_dochorizon', 'extract_with_dochorizon', 'manual__2024-10-05T15:00:07.704358+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/PDF-Klippa.py', '--cfg-path', '/tmp/tmpmjxihopa']
[2024-10-05T15:00:12.635+0000] {standard_task_runner.py:83} INFO - Job 129: Subtask extract_with_dochorizon
[2024-10-05T15:00:12.655+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-05T15:00:12.724+0000] {task_command.py:389} INFO - Running <TaskInstance: pdf_extraction_with_dochorizon.extract_with_dochorizon manual__2024-10-05T15:00:07.704358+00:00 [running]> on host 93d8e18bedbe
[2024-10-05T15:00:12.817+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pdf_extraction_with_dochorizon
AIRFLOW_CTX_TASK_ID=extract_with_dochorizon
AIRFLOW_CTX_EXECUTION_DATE=2024-10-05T15:00:07.704358+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-05T15:00:07.704358+00:00
[2024-10-05T15:00:12.837+0000] {taskinstance.py:1772} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/PDF-Klippa.py", line 62, in extract_with_dochorizon
    encoded_pdf = pdf_file.read().encode('base64')  # Encode file as base64
AttributeError: 'bytes' object has no attribute 'encode'
[2024-10-05T15:00:12.854+0000] {taskinstance.py:1327} INFO - Marking task as UP_FOR_RETRY. dag_id=pdf_extraction_with_dochorizon, task_id=extract_with_dochorizon, execution_date=20241005T150007, start_date=20241005T150012, end_date=20241005T150012
[2024-10-05T15:00:12.868+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 129 for task extract_with_dochorizon ('bytes' object has no attribute 'encode'; 14798)
[2024-10-05T15:00:12.886+0000] {local_task_job.py:159} INFO - Task exited with return code 1
[2024-10-05T15:00:12.921+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
